src/github.com/matrix-org/dendrite/roomserver/storage/sql.go

```diff
@@ -372,6 +394,12 @@ func (s *statements) prepareEvents(db *sql.DB) (err error) {
 	if s.bulkSelectStateEventByIDStmt, err = db.Prepare(bulkSelectStateEventByIDSQL); err != nil {
 		return
 	}
+	if s.bulkSelectStateAtEventByIDStmt, err = db.Prepare(bulkSelectStateAtEventByIDSQL); err != nil {
```

Comment:
From RL: It actually looks surprisingly bad when it's inlined. You end up having to parse SQL and Go together and it really didn't actually make things look easier to read.

We should split up the tables into different files though (but not in this PR).